### DISCUSSION TITLE
Code  Execution  run failed: error: timeout error: signal: killed

### DISCUSSION THREAD

**Discussion by Kevin9703** [langgenius/dify#7274] [Help] [answered]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

If the code execution reports a timeout error after 15 seconds, how can this timeout period be adjusted?
![image](https://github.com/user-attachments/assets/01dfa4bb-9a84-4fd6-af7e-0b236a78be51)

**Comment by tungtk1975** [comment]
I facing the same error, anyone know how to resolve the problem?
Thanks

**Reply by Kevin9703** [reply, marked as answer]
I resolved the timeout issue by adjusting the WORKER_TIMEOUT setting in the sandbox.

**Reply by shengxianli** [reply]
I adjusted the WORKER-TIMEOUT of the. evn and docker-compose.yaml file to 180, but it doesn't seem to work
I am using Docker Compose deployment, what do I need to do after modifying it

**Comment by alexcode2024** [comment]
How can I fixed this problem on a cloud  version dify? @Dosu

**Reply by pingp76** [reply]
same issue on cloud version
